modules/ModuleBuilder/parsers/views/AbstractMetaDataImplementation.php

```diff
@@ -353,12 +353,27 @@ protected function _saveToFile($filename, $defs, $useVariables = true, $forPopup
 
         $out .= ";\n?>\n";
 
-
         if (sugar_file_put_contents($filename, $out) === false) {
             $GLOBALS ['log']->fatal(get_class($this) . ": could not write new viewdef file " . $filename);
         }
     }
 
+    /**
+     * @param $defs The definitions to save
+     * @return bool
+     */
+    private function shouldAppendOriginalViewTemplateDefs($defs)
```

Comment:
Please, rename this function. It sounds like a method for testing purposes. Since it returns boolean, maybe you can use 'is' prefix.

Reply:
Ok done!